firebase-crashlytics/src/test/java/com/google/firebase/crashlytics/internal/model/CrashlyticsReportTest.java

```diff
@@ -226,16 +231,42 @@ private static Application makeTestApplication() {
   }
 
   private static Event makeAnrEvent() {
-    return makeTestEvent("ANR");
+    return Event.builder()
```

Comment:
I see similar create ANR event methods in other unit tests as well. would it be possible to create a utility method to create ANR and use that in all the tests?

Reply:
The codebase currently doesn't use a utility class and there are a decent number of methods that are repeated across unit tests eg.

1. makeTestApplication
2. makeTestEvents

etc.

I can take on a task to refactor this in a separate PR!